### ISSUE TITLE
Tormentors glitch 

### ISSUE THREAD

**Issue by ktha00** [ValveSoftware/Dota2-Gameplay#9065]
### Description

while enemy team hitting tormentors, A glitch of tormentors glows( which reflex dmg glow) near where my current hero is standing or moving.

**Comment by ktha00** [comment]
No, i Mean that glowing thing in showing in screen far from the tormentors, that glowing glitch it is not in the range of tormentors

**Comment by Aluerie** [comment]
Can you make a recording, please. I'm clowning around in demo with this and can't reproduce any glowing glitches.

**Comment by ktha00** [comment]
Here is the video which that the glowing glitch in Tormentors. when Radiance team taking tormentors, this glowing glitch appears next to the Dire team 
https://github.com/ValveSoftware/Dota2-Gameplay/assets/119751906/a69bcae0-aa47-450a-b549-ff48bb51772e

Match id: 7149917711 - 27:11

![Glitch](https://github.com/ValveSoftware/Dota2-Gameplay/assets/119751906/c807c516-f037-4c2d-a806-e0f981f6fa0e)

**Comment by ktha00** [comment]
Your were correct I checked they reply ether I also can't find any of this glitch in reply. This glitch happens in fours match of mine, this one which I recorded for prepose to show. This glitch appears in live matches.
